automation-scripts/prep-new-cluster.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+if ! type helm &>/dev/null ; then
+  curl -sL "https://storage.googleapis.com/kubernetes-helm/helm-v2.6.1-linux-amd64.tar.gz" | tar xz -C /root/bin --strip-components=1 linux-amd64/helm
+  chmod +x /root/bin/helm
+  if kubectl get pods --all-namespaces 2>/dev/null | grep -qi tiller; then
+    echo "Installing helm client"
+    helm init --client-only
+  else
+    echo "Installing helm client and tiller"
+    helm init
+    kubectl create -f - << EOF
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: kube-system:default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: kube-system 
+EOF
+  fi
+fi
+
+if ! type k &>/dev/null ; then
+  echo "Installing k"
+  curl -sLo /root/bin/k "https://github.com/aarondl/kctl/releases/download/v0.0.12/kctl-linux-amd64"
+  chmod +x /root/bin/k
+fi
+
+ 
+if ! type klog.sh &>/dev/null ; then
+  echo "Installing klog"
+  curl -sLo /root/bin/klog.sh "https://raw.githubusercontent.com/SUSE/scf/develop/container-host-files/opt/scf/bin/klog.sh"
+  chmod +x /root/bin/klog.sh
+fi
+
+if ! type cf &>/dev/null; then
+  echo "Installing cf and cf usb"
+  docker run --name cf-usb harts/cf-usb /bin/true
```

Comment:
@HartS i think this should be an official cf-usb. I know we used to have one. If we don't have one currently then @troytop might know why

Reply:
Now that we've released the USB plugin, I'll update this branch to use that one instead

Reply:
@prabalsharma this is down now